src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc

```diff
@@ -95,7 +95,12 @@ Real64 HeatingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
             } else if (this->zoneEqSizing(this->curZoneEqNum).DesignSizeFromParent) {
                 this->autoSizedValue = this->zoneEqSizing(this->curZoneEqNum).DesHeatingLoad;
             } else {
-                if (this->zoneEqSizing(this->curZoneEqNum).HeatingCapacity) {
+                if (this->dataCoilIsSuppHeater && this->suppHeatCap > 0.0) {
+                    NominalCapacityDes = this->suppHeatCap;
+                    if (this->dataFlowUsedForSizing > 0.0) {
+                        DesVolFlow = this->dataFlowUsedForSizing;
+                    }
+                } else if (this->zoneEqSizing(this->curZoneEqNum).HeatingCapacity) {
```

Comment:
UnitarySystem now sets the flag in the HeatingCoils::SimulateHeatingCoilComponents call so that sizing knows this is a supplemental heating coil. With that information UnitarySystem can set the capacity of the coil according to whether CoolReheat dehumidification control is used.